src/Backspace.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Backspace = () => <noscript />;
+
+Backspace.propTypes = {
+  count: PropTypes.number,
+  line: PropTypes.bool,
+  delay: PropTypes.number,
+};
+
+Backspace.defaultProps = {
+  count: 1,
+  line: false,
```

Comment:
What exactly is this prop for? If true, does it mean it will backspace the whole line?

Reply:
maybe we could rename this prop to have a less ambiguous name